app/controllers/catalog_controller.rb

```diff
@@ -41,6 +42,50 @@ def update_user_per_cat_page
     end
   end
 
+  # this method updates item quantities using the edit_reservation_form
+  # rubocop:disable AbcSize
```

Comment:
you can simply put this on the same line as the method definition (and skip the enable line below)